content/coding_standards/ada_coding_standards/chapters/guidelines/dynamic_storage_management/dyn03_access_types_without_allocators_policy.rst

```diff
@@ -0,0 +1,96 @@
+------------------------------------------------
+Access Types Without Allocators Policy (DYN03)
+------------------------------------------------
+
+**Level** :math:`\rightarrow` Advisory
+
+Category
+   :Safety: :math:`\checkmark`
+   :Cyber: :math:`\checkmark`
+
+Goal
+   :Maintainability: :math:`\checkmark`
+   :Reliability: :math:`\checkmark`
+   :Portability:
+   :Performance:
+   :Security: :math:`\checkmark`
+
+**Remediation** :math:`\rightarrow` Low
+
+**Verification Method** :math:`\rightarrow` Compiler restrictions
+
++++++++++++
+Reference
++++++++++++
+
+MISRA C Rule 21.3 "The memory allocation and deallocation functions of <stdlib.h>
+shall not be used"
+
++++++++++++++
+Description
++++++++++++++
+
+The following restrictions must be in effect:
+
+   * No_Allocators
+   * No_Dependence => Ada.Unchecked_Deallocation
+
+In this approach dynamic access values are only created via the attribute
+:ada:`'Access` applied to aliased objects. Allocation and deallocation never
+occur. As a result, storage exhaustion cannot occur because no *dynamic*
+allocations occur. Fragmentation cannot occur because there are no
+deallocations.
+
+In this approach the following constructs are not allowed:
+
+   * Allocators
+   * User-defined storage pools
+   * Unchecked Deallocations
+
+Aspects should be applied to all access types in this approach, specifying a
+value of zero for the storage size.  Although the restriction No_Allocators is
+present, such clauses may be necessary to prevent any default storage pools
+from being allocated for the access types, even though the pools would never be
+used. A direct way to accomplish this is to use pragma Default_Storage_Pool
+with a parameter of :ada:`null` like so:
+
+.. code:: Ada
```

Comment:
@frank-at-adacore the error message is:
```
sphinx.errors.SphinxWarning: /home/runner/work/learn/learn/content/coding_standards/ada_coding_standards/chapters/guidelines/dynamic_storage_management/dyn03_access_types_without_allocators_policy.rst:57:Invalid argument: Ada
```
One difference that I can see looking at the other content here is that the Ada is lowercase, e.g.:
[.. code-block:: ada](https://github.com/AdaCore/learn/blob/a2bf2e4680e13fc7f4aa28c2d1cacf76b95d89de/content/courses/intro-to-ada/chapters/imperative_language.rst#L87)

Reply:
Fixed